build.gradle

```diff
@@ -59,6 +59,7 @@ repositories {
 compileJava {
     options.release = 22
     options.compilerArgs << "--enable-preview"
+    options.compilerArgs << "-Xlint:unchecked"
```

Comment:
Are there many of these? Would be preferrable to fix them in place with `@SuppressWarnings`

Reply:
It was left over from debugging. Added SuppressWarnings as needed. Will remove this.

Reply:
Lines are still there in the latest revision?